README.md

```diff
@@ -39,3 +43,36 @@ Resource group exists:
         organization: EITR Technologies
     - connection_auth: {{ profile }}
 ```
+
+# Security
+
+To make the file more secure, use the acct comand to encrpt the file with the Fernet algorithm
+
+~~~
```

Comment:
code blocks are designated with a "back tick". I think you've inserted tildes. Don't hold down "shift".